internal/cli/opt/opt.go

```diff
@@ -105,3 +105,11 @@ func (o *ProjectOption) Complete() error {
 func AddProjectFlags(cmd *cobra.Command, o *ProjectOption) {
 	cmd.Flags().StringVarP(&o.Project, "project", "p", o.Project, "If present, the project scope for this CLI request")
 }
+
+type ForceCreateOption struct {
```

Comment:
Do you see an usage of this flag in other command ? If no, probably it's not worth it to create a struct to handle this flag

Reply:
removed in 7c25ff5396abc6515bf8fef305b78c5312981481